catalog/app/containers/Bucket/PackageCreateDialog.js

```diff
@@ -353,6 +356,10 @@ const getTotalProgress = R.pipe(
 
 const defaultNameWarning = ' ' // Reserve space for warning
 
+const selectUser = createStructuredSelector({
```

Comment:
you dont need a structured selector here, you can just use `authSelectors.username` directly

Reply:
done